From 7602bd4138d78c7df9479b3f760895cecd10d7d9 Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Tue, 16 Dec 2014 16:10:05 -0500
Subject: [PATCH 51/53] Ticket 47451 - Fix jenkins errors

Description:  Fix compiler warnings

https://fedorahosted.org/389/ticket/47451
(cherry picked from commit 3e1d97624f390b96c1de21b115619822b72ebf98)
(cherry picked from commit 15e1cdd7d01933d42b83b00e1e8c01c16fd36e8e)
---
 ldap/servers/plugins/dna/dna.c     | 6 +-----
 ldap/servers/plugins/uiduniq/uid.c | 2 --
 2 files changed, 1 insertion(+), 7 deletions(-)

diff --git a/ldap/servers/plugins/dna/dna.c b/ldap/servers/plugins/dna/dna.c
index ded0bbb..f4a36f0 100644
--- a/ldap/servers/plugins/dna/dna.c
+++ b/ldap/servers/plugins/dna/dna.c
@@ -2406,7 +2406,7 @@ static int dna_get_next_value(struct configEntry *config_entry,
         } else {
             /* dna_first_free_value() failed for some unknown reason */
             slapi_log_error(SLAPI_LOG_FATAL, DNA_PLUGIN_SUBSYSTEM,
-                            "dna_get_next_value: failed to allocate a new ID!! (set(%d) (max: %d)\n",setval,config_entry->maxval);
+                            "dna_get_next_value: failed to allocate a new ID!!\n");
             goto done;
         }
     }
@@ -3415,8 +3415,6 @@ _dna_pre_op_add(Slapi_PBlock *pb, Slapi_Entry *e, char **errstr)
                         /* dna_first_free_value() failed for some unknown reason */
                         slapi_log_error(SLAPI_LOG_FATAL, DNA_PLUGIN_SUBSYSTEM,
                                         "dna_pre_op: failed to allocate a new ID!! 2\n");
-                        slapi_log_error(SLAPI_LOG_FATAL, DNA_PLUGIN_SUBSYSTEM,
-                                                    "dna_get_next_value: failed to allocate a new ID!! (set(%d) (max: %d)\n",setval,config_entry->maxval);
                         /* Set an error string to be returned to the client. */
                         *errstr = slapi_ch_smprintf("Allocation of a new value for range"
                                                " %s failed! Unable to proceed.",
@@ -3682,8 +3680,6 @@ _dna_pre_op_modify(Slapi_PBlock *pb, Slapi_Entry *e, Slapi_Mods *smods, char **e
                         /* dna_first_free_value() failed for some unknown reason */
                         slapi_log_error(SLAPI_LOG_FATAL, DNA_PLUGIN_SUBSYSTEM,
                                         "dna_pre_op: failed to allocate a new ID!!\n");
-                        slapi_log_error(SLAPI_LOG_FATAL, DNA_PLUGIN_SUBSYSTEM,
-                                                    "dna_get_next_value: failed to allocate a new ID!! (set(%d) (max: %d)\n",setval,config_entry->maxval);
                         /* Set an error string to be returned to the client. */
                         *errstr = slapi_ch_smprintf("Allocation of a new value for range"
                                            " %s failed! Unable to proceed.",
diff --git a/ldap/servers/plugins/uiduniq/uid.c b/ldap/servers/plugins/uiduniq/uid.c
index f37ab8c..2120b4f 100644
--- a/ldap/servers/plugins/uiduniq/uid.c
+++ b/ldap/servers/plugins/uiduniq/uid.c
@@ -1285,7 +1285,6 @@ uiduniq_start(Slapi_PBlock *pb)
 static int
 uiduniq_close(Slapi_PBlock *pb)
 {
-    Slapi_Entry *plugin_entry = NULL;
     struct attr_uniqueness_config *config = NULL;
 
     slapi_pblock_get(pb, SLAPI_PLUGIN_PRIVATE, &config);
@@ -1312,7 +1311,6 @@ NSUniqueAttr_Init(Slapi_PBlock *pb)
   int preadd = SLAPI_PLUGIN_PRE_ADD_FN;
   int premod = SLAPI_PLUGIN_PRE_MODIFY_FN;
   int premdn = SLAPI_PLUGIN_PRE_MODRDN_FN;
-  struct attr_uniqueness_config *config = NULL;
 
   BEGIN
 
-- 
1.9.3

